ui/src/app/pages/analysis/new-notebook-modal.tsx

```diff
@@ -33,7 +34,7 @@ export const NewNotebookModal = (props: Props) => {
   const [nameTouched, setNameTouched] = useState(false);
   const [navigate] = useNavigation();
 
-  const { workspace, onClose, existingNameList } = props;
```

Comment:
These files should not be here - something has gone wrong.  Could you fetch and rebase against the latest `main` ?  I can pair on this.

Reply:
Good call. I merged main into the branch, so I believe that this should be fixed now.